packages/botonic-core/src/pusher-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { inflate } from 'pako/dist/pako_inflate.js'
+
+/**
+ *
+ * @param compressedData a string representing gzipped data previously encoded to base64 for very large contents
+ * @returns a string representing the information of a very large content
+ * Ref: https://stackoverflow.com/questions/4875020/javascript-decompress-inflate-unzip-ungzip-strings
+ */
+export function decompressData(compressedData) {
+  const strData = atob(compressedData)
```

Comment:
very extra: should `compressedData` be typed? to understand better this atob?

Reply:
added types and clarifications! thanks for the feedback G =)